src/app/util/BUILD.gn

```diff
@@ -75,3 +75,11 @@ source_set("callbacks") {
     "${chip_root}/src/app:paths",
   ]
 }
+
+source_set("binding-table-headers") {
+  sources = [ "binding-table.h" ]
+}
+
+source_set("ember-api-headers") {
+  sources = [ "ember-api.h" ]
```

Comment:
we should not have a header without the implementation cpp unless that header defines constants only. I.e. if I depend on `foo` I should be able to link. This breaks that.